packages/botonic-react/tests/webchat/attachments.test.js

```diff
@@ -0,0 +1,58 @@
+import {
+  isAllowedSize,
+  getAttachmentType,
+  getAcceptedFormats,
+} from '../../src/message-utils'
+
+describe('TEST: attachments ', () => {
+  const createFile = ({ fakeData, fileName, mimeType, sizeInMB }) => {
+    const file = new File([fakeData], fileName, { type: mimeType })
+    if (sizeInMB) {
+      // https://stackoverflow.com/a/55638956
+      Object.defineProperty(file, 'size', {
+        value: sizeInMB * 1024 * 1024,
+        writable: false,
+      })
+    }
+    return file
+  }
+
+  it('Gets correctly the attachment type', () => {
+    const file = createFile({
```

Comment:
don't create the file if you don't use it

Reply:
my idea was to also cover the same way as the File will be called in a browser, is really that wrong to have it like this?